### PR TITLE
Removes move sound from chap armor

### PR DESCRIPTION
**Pull request by Sinmero**
## About The Pull Request

Chap armor move sound is loud and obnoxious. 
You are forced to hear it ALL the time when you walk.
Removes move sound from chap armor.

## Why It's Good For The Game

Your ears are safe now.

## Changelog
:cl:
del: Removes move sound from chap armor
/:cl:

### CODE REVIEW

**Comment by ItsTisme** [comment]
I think riot armor is fine but for chaps its pretty annoying yeah

**Comment by Sinmero** [comment]
>I think riot armor is fine but for chaps its pretty annoying yeah

I think that the sound is bad enough that it is punishing to wear those items.

**Comment by r1ks-iwnl** [comment]
> 
> 
> > I think riot armor is fine but for chaps its pretty annoying yeah
> 
> I think that the sound is bad enough that it is punishing to wear those items.

Being able to hear a seccie patrolling maint with riot armor and a shotgun seems like a good ideea to me.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Remove from chaplains, keep for riot armour

**Comment by Sinmero** [comment]
> Remove from chaplains, keep for riot armour

Done

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -27,6 +27,7 @@
 	allowed = list(/obj/item/storage/book/bible, /obj/item/nullrod, /obj/item/reagent_containers/food/drinks/bottle/holywater, /obj/item/storage/fancy/candle_box, /obj/item/candle, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
 	slowdown = 0
 	blocks_shove_knockdown = FALSE
+	move_sound = list()
```
Comment (PowerfulBacon) [inline code/game/objects/items/holy_weapons.dm]:
```suggestion
	move_sound = null
```

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

File: code/game/objects/items/holy_weapons.dm (hunk 1 above)
Comment (Sinmero) [reply code/game/objects/items/holy_weapons.dm]: done

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**